docs/source/advanced/java-rxjava.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: 'Java and RxJava support'
+---
+
+import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
+
+
+Because Apollo Android is written in Kotlin, it can be used in Java projects. This guide will show you how to do that.
+
+## Use the Java codegen
+
+Apollo Android generates Kotlin code by default, but you can configure it to use the Java codegen instead:
+
+```kotlin:title=build.gradle[.kts]
+apollo {
+  generateKotlinModels.set(false)
+}
+```
+
+## Use RxJava adapters
```

Comment:
I'd try to stay clear of "adapters" here as it's pretty loaded already. Maybe "RxJava extensions"?

Reply:
👍 agreed